src/analysis/normed_space/int.lean

```diff
@@ -0,0 +1,39 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import analysis.normed_space.basic
+
+/-!
+# The integers as normed ring
+
+This file contains basic facts about the integers as normed ring
+-/
```

Comment:
Maybe it's worth saying something about what ||x|| and ||x||_+ are? I guess the latter is quite new.